plone/app/locales/locales/fr/LC_MESSAGES/plone.po

```diff
@@ -8847,7 +8847,7 @@ msgstr "Votre demande de réinitialisation de mot de passe vous a été envoyée
 #. Default: "Move the translation under another language folder"
 #: plone/app/multilingual/browser/menu.py:141
 msgid "description_set_language"
-msgstr "Préciser ou modifier la langue actuelle du contenu"
+msgstr "Saisissez ou modifiez la langue actuelle du contenu"
```

Comment:
Yeah, I don't think it's a input for this one, so I would prefer "Spécifiez" here.